src/inference/tests/unit/query_model_test.cpp

```diff
@@ -461,3 +463,30 @@ TEST_F(GetSupportedNodesTest, ShuffleChannelFusion) {
         },
         {});  // Nothing is supported due to unsupported ShuffleChannels
 }
+
+TEST_F(GetSupportedNodesTest, FusedNameReduceL2Test) {
+    {
+        auto data = std::make_shared<ov::opset9::Parameter>(ov::element::f32, ov::Shape{1, 512});
+        data->set_friendly_name("data");
+        auto axes = ov::opset9::Constant::create(ov::element::i64, ov::Shape{1}, {1});
+        axes->set_friendly_name("axes");
+        auto reduce_l2 = std::make_shared<ov::opset9::ReduceL2>(data, axes, true);
+        reduce_l2->set_friendly_name("reduce_l2");
+
+        m_function = std::make_shared<ngraph::Function>(ov::NodeVector{reduce_l2}, ov::ParameterVector{data});
+    }
+    Run(
+        [&](std::shared_ptr<ov::Model>& model) {
+            ov::pass::Manager m;
+            m.register_pass<ov::pass::InitNodeInfo>();
+            m.register_pass<ov::pass::ReduceL2Decomposition>();
+            m.register_pass<ov::pass::ConvertReduceToPooling>();
+            m.run_passes(model);
+        },
+        [&](const std::shared_ptr<ngraph::Node>& op) {
+            // Pooling is supported, but Sqrt is not
+            return ov::op::util::is_parameter(op) || ov::op::util::is_output(op) || ov::op::util::is_constant(op) ||
+                   (std::dynamic_pointer_cast<ov::opset1::AvgPool>(op) != nullptr);
+        },
+        {});  // Check that constant axis is removed from supported
```

Comment:
again, why do we need a test for QN?
The bug in transformations and we need a test in `transformations` folder.

Reply:
> again, why do we need a test for QN? The bug in transformations and we need a test in `transformations` folder.

Transformation tests check that RT info is located inside operations. And it is actually located. The question what exactly located inside. This bug is reproduced in combinations of two sunsequent transformations and in condition when only part of new transformed graph is supported by plugin. Any improvements for tranformations implementatios and tests are welcome, but it should not be part of bugfix.

Reply:
> The question what exactly located inside

@itikhono can we improve tests for transformations to check rt_info more strictly?

Reply:
@t-jankowski could you take a look?

Reply:
As a hot fix it's ok to place it here (considering suggestion I put in the other comment). However actual solution should be either in `ngraph::copy_runtime_info` or transformations dedicated wrapper function.
As of the transformations copying runtime info test - yes, it should be improved.